src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -22,7 +22,7 @@
  */
 class DoctrineChoiceLoader extends AbstractChoiceLoader
 {
-    private ObjectManager $manager;
+    private $manager;
     private string $class;
     private ?IdReader $idReader;
```

Comment:
Nullable properties are not concerned? :thinking:

Reply:
all types are concerned